optuna_dashboard/_app.py

```diff
@@ -331,6 +332,23 @@ def save_trial_user_attrs(trial_id: int) -> dict[str, Any]:
         response.status = 204
         return {}
 
+    @app.post("/api/studies/<study_id:int>/<trial_id:int>/skip")
+    @json_api_view
+    def skip_trial(study_id: int, trial_id: int) -> dict[str, Any]:
+        summary = get_study_summary(storage, study_id)
+        if summary is None:
+            response.status = 404  # Not found
+            return {"reason": f"study_id={study_id} is not found"}
+        system_attrs = getattr(summary, "system_attrs", {})
```

Comment:
Here, you can simply call `get_study_system_attrs()` storage api.
https://github.com/optuna/optuna/blob/ddfcafd796eb7f933438510fdc3c844864933fe9/optuna/storages/_base.py#L205

```suggestion
        try:
            system_attrs = storage.get_study_system_attrs(study_id)
        except KeyError:
            response.status = 404  # Not found
            return {"reason": f"study_id={study_id} is not found"}
```